resource/ui/ts/models/timeseries.ts

```diff
@@ -43,68 +44,183 @@ module Models {
         /**
          * QueryResultSet matches the successful output of the /ts/query
          * endpoint. This needs to be kept in sync with the
-         * TimeSeriesQueryResult protobuffer message on the server.
+         * TimeSeriesQueryResponse protobuffer message on the server.
          * (/protos/timeseries.proto).
          */
         export interface QueryResultSet {
```

Comment:
That's an actual protocol buffer implementation - as in, packed bytes on the wire instead of JSON. It also seems to be a pretty monstrous library, AND it also doesn't appear to generate typescript code.

I wouldn't be opposed to a library that generated typescript definitions based on the Json representation of protobuffers, if such a thing exists.